### PR TITLE
Feature/present option for attributes

### PR DESCRIPTION
**Pull request by nesaulov**
Hi!
Firstly, thank you for this gem, i find it really great and would like to contribute my work to it.

I've implemented `present:` option that is mentioned in #4

So now one can specify, for example:
``` ruby
attribute :name, String, present: true
```
And error will be raised if this attribute is not provided.

Also i've improved some typos in documentation and have bumped version if you don't mind  :)

Please let me know what do you think of these changes.

Have a great day!

### CODE REVIEW

File: lib/shallow_attributes/version.rb [hunk 1]
```diff
@@ -2,5 +2,5 @@ module ShallowAttributes
   # Defines the full version
   #
   # @since 0.1.0
-  VERSION = "0.9.2".freeze
+  VERSION = "0.10.0".freeze
```
Comment (davydovanton) [inline lib/shallow_attributes/version.rb]: Could you revert this change, please? :)

**Comment by nesaulov** [comment]
Surely, i've reverted this.

**Comment by nesaulov** [comment]
And updated the README a bit (added link to mandatory-attributes section)
